32.md

```diff
@@ -0,0 +1,172 @@
+NIP-32
+======
+
+Labeling
+---------
+
+`draft` `optional` `author:staab` `author:gruruya` `author:s3x-jay`
+
+A label is a `kind 1985` event that is used to label other entities. This supports a number of use cases:
+
+- Distributed moderation and content recommendations
+- Reviews and ratings
+- Definition of edges in a graph structure
+
+This NIP does not supersede NIP-56, which supports reporting content for the purpose of
```

Comment:
I mostly wanted to avoid pushback based on adding a second way to do the same thing. I agree this paragraph is sort of BS, but it seems worth keeping.

Reply:
Alright, I've removed it, we'll leave it up to people to interpret.